models/attribute-type/src/attribute-localized-text-type/builder.spec.ts

```diff
@@ -0,0 +1,29 @@
+/* eslint-disable jest/no-disabled-tests */
+/* eslint-disable jest/valid-title */
+import { createBuilderSpec } from '@commercetools-test-data/core/test-utils';
+import { type TAttributeLocalizedString } from './types';
+import * as AttributeLocalizedTextType from './index';
+
+describe('builder', () => {
+  it(
+    ...createBuilderSpec<TAttributeLocalizedString, TAttributeLocalizedString>(
+      'default',
+      AttributeLocalizedTextType.random(),
+      expect.objectContaining({
+        name: 'ltext',
+      })
+    )
+  );
+
+  it(
+    ...createBuilderSpec<TAttributeLocalizedString, TAttributeLocalizedString>(
+      'rest',
+      AttributeLocalizedTextType.random(),
+      expect.objectContaining({
+        name: 'ltext',
+      })
+    )
+  );
+
+  //There are no graphql tests at this time
```

Comment:
For completeness, we could use a simple test along the lines of [this one](https://github.com/commercetools/test-data/blob/0d102a540f7c2fe462f54f0e67e8b2d869d3bdc4/models/attribute-type/src/attribute-boolean-type/attribute-boolean-type-draft/builder.spec.ts#L31), but it's such a simple model. Your call 😄